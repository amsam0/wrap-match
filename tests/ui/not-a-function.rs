fn main() {}

#[wrap_match::wrap_match]
struct NotAFunction;
