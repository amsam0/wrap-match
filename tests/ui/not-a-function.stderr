error: expected `fn`
 --> tests/ui/not-a-function.rs:4:1
  |
4 | struct NotAFunction;
  | ^^^^^^
